cli/factory.go

```diff
@@ -96,3 +97,37 @@ var destroyFactoryCommand = cli.Command{
 		return nil
 	},
 }
+
+var statusFactoryCommand = cli.Command{
+	Name:  "status",
+	Usage: "query the status of VM factory",
+	Action: func(context *cli.Context) error {
+		runtimeConfig, ok := context.App.Metadata["runtimeConfig"].(oci.RuntimeConfig)
+		if !ok {
+			return errors.New("invalid runtime config")
+		}
+
+		if runtimeConfig.FactoryConfig.Template {
+			factoryConfig := vf.Config{
+				Template: true,
+				VMConfig: vc.VMConfig{
+					HypervisorType:   runtimeConfig.HypervisorType,
+					HypervisorConfig: runtimeConfig.HypervisorConfig,
+					AgentType:        runtimeConfig.AgentType,
+					AgentConfig:      runtimeConfig.AgentConfig,
+				},
+			}
+			kataLog.WithField("factory", factoryConfig).Info("load vm factory")
+			f, err := vf.NewFactory(factoryConfig, true)
+			if err != nil {
+				fmt.Println("vm factory is off")
```

Comment:
It might be worth changing this (and the other functions in this file) to be something like:

```
fmt.Fprintln(defaultOutputFile, "vm factory is off")
```

Reply:
Done.

Reply:
Thanks!